pkg/operator/k8sutil/network_test.go

```diff
@@ -53,6 +80,9 @@ func TestNetwork_ApplyMultusJSON(t *testing.T) {
 	}
 
 	objMeta := metav1.ObjectMeta{}
+	objMeta.Labels = map[string]string{
+		"app": "rook-ceph-osd",
+	}
 	err := ApplyMultus(net, &objMeta)
 	assert.NoError(t, err)
 
```

Comment:
I realize the  test below is pre-existing, but my previous comment still applies. It is better to test that the annotations are exactly as desired and not merely that they "contain" the things desired because the function could be modified in the future to add undesirable content. It would be good to modify the below to check that the annotation content is exactly as desired or at minimum verify that there are only 2 items present under `objMeta.Annotations["k8s.v1.cni.cncf.io/networks"]`